### PR TITLE
Update README.md for addition of logo

### PR DESCRIPTION
**Pull request by ruddi10**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -75,21 +75,21 @@ The above block will render the following:
...
 
 7. Replace your résumé with `resume.pdf` in `src/assets/` and make sure to
-rename it to `resume.pdf`.
+   rename it to `resume.pdf`.
 
 8. If you would like to add your preferred tech-stack icons', please find their
-SVGs online and add it to `src/assets/icon/` directory. After that, simply
-import the icon in `src/components/skills/index.tsx` and append it to the
-`LOGOS` array.
+   SVGs online and add it to `src/assets/icon/` directory. After that, open the svg image in the code editor and give it an id attribute `tech` and then simply
```
Comment (algomaster99) [inline README.md:88]: Make sure you restrict this to 80 characters.

File: README.md [hunk 2]
```diff
@@ -18,45 +18,45 @@ The development and production processes are automated through
...
 3. Run `npm start` to start the development server at `http://localhost:3000`.
 
 4. You may use `npm run code-format` and `npm run style-format` to format the
-TypeScript and SASS code respectively.
+   TypeScript and SASS code respectively.
 
 5. Run `npm run build` to build the static page for deployment.
 
-## $ chown <your name> portfolio
+## \$ chown <your name> portfolio
```
Comment (algomaster99) [inline README.md:41]: Why is a `\` added here?

**Review by algomaster99 — CHANGES_REQUESTED** [review]
Thanks for your PR :)

**Comment by ruddi10** [comment]
Thanks for the review made the changes. PTAL

File: README.md [hunk 3]
```diff
@@ -75,21 +75,22 @@ The above block will render the following:
...
-rename it to `resume.pdf`.
+   rename it to `resume.pdf`.
 
 8. If you would like to add your preferred tech-stack icons', please find their
-SVGs online and add it to `src/assets/icon/` directory. After that, simply
-import the icon in `src/components/skills/index.tsx` and append it to the
-`LOGOS` array.
+   SVGs online and add it to `src/assets/icon/` directory. After that, open the
+   svg image in the code editor and give it an id attribute `tech` and then simply
+   import the icon in `src/components/skills/index.tsx` and append it to the
```
Comment (algomaster99) [inline README.md:90]: icon in `src/components/skills/index.tsx`. Finally, append it to the ...

**Review by algomaster99 — CHANGES_REQUESTED** [review]
Sorry, forgot to mark this as a change. This is the last one :)

**Comment by ruddi10** [comment]
PTAL

**Review by algomaster99 — APPROVED** [review]

**Pull request merged**